src/components/skills-quiz/__mocks__/react-instantsearch-dom.jsx

```diff
@@ -6,40 +6,52 @@ const MockReactInstantSearch = jest.genMockFromModule(
   'react-instantsearch-dom',
 );
 
-const fakeHits = [
-  {
-    objectID: '1',
-    title: 'bla',
-    key: 'Bees101',
-    type: 'course',
-    aggregation_key: 'course:Bees101',
-    authoring_organizations: [],
-    card_image_url: 'https://fake.image',
-    course_keys: [],
-  },
-  {
-    objectID: '2',
-    title: 'blp',
-    key: 'Wasps200',
-    type: 'course',
-    aggregation_key: 'course:Wasps200',
-    authoring_organizations: [],
-    card_image_url: 'https://fake.image',
-    course_keys: [],
-  },
-];
-
-MockReactInstantSearch.configure = {
+let mockState = {
+  hits: [
+    {
+      objectID: '1',
+      title: 'bla',
+      key: 'Bees101',
+      type: 'course',
+      aggregation_key: 'course:Bees101',
+      authoring_organizations: [],
+      card_image_url: 'https://fake.image',
+      course_keys: [],
+    },
+    {
+      objectID: '2',
+      title: 'blp',
+      key: 'Wasps200',
+      type: 'course',
+      aggregation_key: 'course:Wasps200',
+      authoring_organizations: [],
+      card_image_url: 'https://fake.image',
+      course_keys: [],
+    },
+  ],
   nbHits: 2,
 };
 
+// This allows you to override the built-in hits object
+const setFakeHits = hits => {
+  mockState.hits = hits;
+  mockState.nbHits = hits.length;
+};
+
+const resetMockReactInstantSearch = () => {
+  mockState = {
+    hits: [...mockState.hits], // deep clone if needed
```

Comment:
[curious] If a test calls `setFakeHits` to override `mockState.hits`, and then calls `resetMockReactInstantSearch`, wouldn't the `mockState.hits` you're spreading here represent the modified `hits`, not the original `hits`?

Should this keep track of the original fake hits and have the reset function revert back to the original vs. the potentially modified hits? E.g.,

```jsx
const originalFakeHits = [{ ... }, { ... }];
const mockState = {
  hits: originalFakeHits,
  nbHits: originalFakeHits.length,
};
const setFakeHits = hits => {
  mockState.hits = hits;
  mockState.nbHits = hits.length;
};
const resetMockReactInstantSearch = () => {
  mockState.hits = originalFakeHits;
  mockState.nbHits = originalFakeHits.length;
};
```